storage/invoke.go

```diff
@@ -64,9 +64,9 @@ func shouldRetry(err error) bool {
 	}
 	switch e := err.(type) {
 	case *googleapi.Error:
-		// Retry on 429 and 5xx, according to
+		// Retry on 408, 429, and 5xx, according to
```

Comment:
Please update the docs for WithErrorFunc as well to clarify the new default. https://pkg.go.dev/cloud.google.com/go/storage@main#WithErrorFunc